extras/installer.sh

```diff
@@ -277,6 +277,9 @@ fi
 
 echo
 read -e -p "Directory to install into: " -i "/opt/plexupdate" FULL_PATH
+if [[ "$FULL_PATH" == *"~"* ]]; then # If path contains a tilde
```

Comment:
That's because `~user` isn't an option, it's an abomination. :)

Short of writing a stupidly complicated function just for this use case (or using `eval` which is terrible for user-entered data), I think the best we can do is something like:

FULL_PATH=${FULL_PATH/\~\//$HOME}

It'll still break for `~user`, but at least it'll keep the info as entered by them so they won't be confused by an error message.